src/gui/Base/HTile.qml

```diff
@@ -110,12 +110,21 @@ HButton {
 
     TapHandler {
         acceptedButtons: Qt.RightButton
+        acceptedPointerTypes: PointerDevice.GenericPointer
```

Comment:
`PointerDevice.Pen` should be accepted here too, some have physical buttons (tested with a wacom tablet pen)

Reply:
Then we have to choose, either here or in long press? Will wait for your reply

Reply:
I guess both are fine? I don't think there's a way to know if the user's pen has a button or not, but having the menu on long-tap anyway shouldn't be too much of a bother.

Reply:
Indeed.